applications/README.md

```diff
@@ -0,0 +1,25 @@
+# Applications
+This directory contains CDK-based serverless applications.
+
+## Prerequisites
+
+* Either install esbuild (`npm install -g esbuild`); otherwise the build process will default to using Docker images which requires Docker to be installed and running locally. Note that esbuild is _much_ faster than Docker
+* You may need to run `cdk bootstrap` before provisioning infrastructure
+
+## Useful commands
+
+You may need to run `source ../parameters.sh` beforehand.
+
+* `npm run build`   compile typescript to js
+* `npm run watch`   watch for changes and compile
+* `npm run test`    perform the jest unit tests
+* `npx cdk deploy`  deploy this stack to your default AWS account/region
+* `npx cdk diff`    compare deployed stack with current state
+* `npx cdk synth`   emits the synthesized CloudFormation template
+
```

Comment:
Would be good to add a section on testing, especially testing locally (and/or via LocalStack).

Reply:
Yep good shout, I've added below